src/channel.ts

```diff
@@ -1166,6 +1167,27 @@ export class Channel {
     });
   }
 
+  /**
+   * markDelivered - Send the mark delivered event for this user, only works if the `delivery_receipts` setting is enabled
+   *
+   * @param {MarkDeliveredOptions} data
+   * @return {Promise<EventAPIResponse | null>} Description
+   */
+  async markDelivered(data: MarkDeliveredOptions) {
```

Comment:
@rafaelmf3 I will be moving this to the client as we need to build a payload across multiple channels - we need to be able to mark multiple channels as delivered with one request